### ISSUE TITLE
Makefile:2: recipe for target 'install' failed

### ISSUE THREAD

**Issue by jxv** [mfine/heroku-buildpack-stack#26]
Hi, I've been using your project for several months, and it's been a lifesaver. Haskell in the cloud!
Unfortunately, my project doesn't build anymore. It just stumbled on this today. I'm not even sure if it's me. Perhaps, you can double check with your apps.
I'm hoping it's a cache issue, but I haven't made any Heroku related changes since August.
 
Makefile:
```
install:
	    stack build --copy-bins
```

Build/deploy log:
```
-----> Haskell app detected
-----> 2017-10-25_00:59:47 Starting...
-----> 2017-10-25_00:59:47 Exporting config vars
-----> 2017-10-25_00:59:47 Restoring 6.0.0a files from cache
-----> 2017-10-25_00:59:47 Restoring stack-1.5.1
-----> 2017-10-25_00:59:47 Substituting stack vars
-----> 2017-10-25_00:59:47 Restoring .stack-work
-----> 2017-10-25_00:59:51 Running stack
-----> 2017-10-25_00:59:51 Makefile detected
stack build --copy-bins
Process exited with ExitFailure 1: /usr/bin/gcc -v
Makefile:2: recipe for target 'install' failed
make: *** [install] Error 1
 !     Push rejected, failed to compile Haskell app.
 !     Push failed
```

**Comment by mfine** [comment]
Can you provide a lot more context? Could you reduce the problem down to something that's reproduceable that you could share? Thanks!

**Comment by jxv** [comment]
Good news! Sorry, for the delay. It was user error, a strange one too.

While failing to reproduce the same error in another project, I realized that I was running on a nightly stackage version from May. I bumped that. No avail. I still wasn't sure if heroku was at fault or what I should isolate. Turned on verbose. Cleared the cache. And slowly toggled away heroku settings. No change.

The problem was already staring me in the face, but I couldn't test on OSX. Turns out `/usr/bin/gcc -v` will read a file called `specs` if it exists. I just created a directory named `specs`, and it was failing to read that. Re-renamed that directory, then deployed it successfully.
